lib/sensu-mutator.rb

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env ruby
+#
+# Sensu-mutator
+# ===
+#
+# DESCRIPTION:
+#   Base mutator class.  All you need to do is extend this class and implement a
+#   #mutate function.
+# 
+#  trival case as described https://sensuapp.org/docs/latest/mutators#example-mutator-plugin:
```

Comment:
An you elaborate on the 'old approach'?  There really isn't one. If you are referring to how the PR on (sensu-plugins-pagerduty)[https://github.com/sensu-plugins/sensu-plugins-pagerduty/pull/12].  I changed here since the standard of using autorun was already defined.